fix for "error: shared_ptr in namespace std does not name a template type"
author: Kenneth Hoste (HPC-UGent)
--- satsuma2-37c5f386819614cd3ce96016b423ddc4df1d86ec/analysis/SatsumaSynteny2.cc.orig	2022-08-22 09:58:49.690714992 +0200
+++ satsuma2-37c5f386819614cd3ce96016b423ddc4df1d86ec/analysis/SatsumaSynteny2.cc	2022-08-22 09:59:05.103870557 +0200
@@ -7,6 +7,7 @@
 
 #include <string.h>
 #include <unistd.h>
+#include <memory>
 #include "../base/CommandLineParser.h"
 #include "SequenceMatch.h"
 #include "GridSearch.h"
